File: Example/ios/iOS Example/EventsViewController.m
#import "EventsViewController.h"
#import "HubjsTracker.h"


@interface EventsViewController ()
@end


@implementation EventsViewController


- (void)viewDidLoad {
  [super viewDidLoad];
  
  [[HubjsTracker sharedInstance] sendViews:@"menu", @"events", nil];
}


- (IBAction)sendEventAction:(id)sender {
  
  // Send a custom event tp Hubjs  
  [[HubjsTracker sharedInstance] sendEventWithCategory:@"TestCategory" action:@"Play" name:@"Song12" value:@(2)];
  
}


@end

File: Example/ios/iOS Example/EventsViewController.swift
import UIKit
import HubjsTracker

class EventsViewController: UIViewController {
    override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(animated)
        HubjsTracker.shared.track(view: ["menu","events"])
    }
    @IBAction func trackEventButtonTapped(_ sender: UIButton) {
        HubjsTracker.shared.track(eventWithCategory: "TestCategory", action: "TestAction", name: "TestName", value: 7)
    }
}
